pkg/mls/store/store.go

```diff
@@ -125,33 +134,32 @@ func (s *Store) PublishIdentityUpdate(ctx context.Context, req *identity.Publish
 			}
 			updates = append(updates, identityUpdate)
 		}
-		_ = append(updates, new_update)
+		_ = append(updates, newUpdate)
 
-		state, err := validationService.GetAssociationState(ctx, updates, []*associations.IdentityUpdate{new_update})
+		state, err := validationService.GetAssociationState(ctx, updates, []*associations.IdentityUpdate{newUpdate})
```

Comment:
Not directly related to the isolation issues, but isn't this a bug? The signature of `GetAssociationState` expects `ctx, oldUpdates, newUpdates`, but in this case the new update is being placed in both the `oldUpdates` and `newUpdates` param